livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py

```diff
@@ -274,8 +280,11 @@ async def recv_task(ws: aiohttp.ClientWebSocketResponse):
                     if closing_ws:  # close is expected, see SpeechStream.aclose
                         return
 
-                    raise Exception(
+                    raise APIStatusError(
                         "AssemblyAI connection closed unexpectedly",
+                        status_code=-1,
+                        request_id=None,
+                        body=None,
```

Comment:
just style nit: probably leave these out? instead of setting explicitly to None

Reply:
its for typecheck

Reply:
ah I see. should we fix the constructor to default to None for them? and maybe we can pass in a `request_id` when available

Reply:
![image](https://github.com/user-attachments/assets/63567f2c-cddb-4bd4-af5d-19cc6cd51586)
it expects those params, or we can adjust defination to allow raising APIStatusError without explicitly setting each param

Reply:
yes.. that's what I meant.. can't we just change the class so it's not as annoying to use?

Reply:
anyways, just a nit. the rest of the change looks good. let's get a fix out as this would improve recoverability

Reply:
updated, Let me know if looks good

Reply:
lgtm